SUMMARY.md

```diff
@@ -0,0 +1,23 @@
+# Table of contents
+
+* [Hello Friend!](README.md)
+
+## Read
+
+* [BOLT \#0: Introduction](read/00-introduction.md)
+* [BOLT \#1: Messaging](read/01-messaging.md)
+* [BOLT \#2: Peer Protocol](read/02-peer-protocol.md)
+* [BOLT \#3: Transactions](read/03-transactions.md)
+* [BOLT \#4: Onion Routing](read/04-onion-routing.md)
+* [BOLT \#5: On-chain](read/05-onchain.md)
+* [BOLT \#6: DEPRECATED](https://github.com/lightningnetwork/lightning-rfc/issues/551)
```

Comment:
imo we don't need to display BOLT 6, but we can keep if others feel strongly

Reply:
fwiw i really like having the explanation for its exclusion linked, much like the "this page intentionally left blank" notes on legal docs.

Reply:
Same!

Reply:
I'd also vouch for keeping this since the GitBook site is rather targeting spec newbies (who I bet will ask themselves what happened to BOLT #6 - at least I did).